src/status_im/subs/root.cljs

```diff
@@ -55,6 +55,7 @@
 (reg-root-key-sub :get-pairing-installations :pairing/installations)
 (reg-root-key-sub :tooltips :tooltips)
 (reg-root-key-sub :biometric/supported-type :biometric/supported-type)
+(reg-root-key-sub :biometric/auth-pending? :biometric/auth-pending?)
```

Comment:
Do we need to add another root key to the db?
Cc @ulisesmac - maybe there's a better approach :)

Reply:
Maybe it's time to make a root key for biometric and put `auth-method`, `supported-type`, `auth-pending?` there? _(not sure if there are others)_

Reply:
@clauxx 

I believe it's better having a single root key for `:biometric` and inside creating a map to handle all the related data, then extract the data using derived subscriptions that depend on the root `:biometrics`.

The benefits are mainly code organization, it's easier to extract `(:biometric db)` to get all the related data than:
`(select-keys db [:biometric/auth-pending? :bio.../supported-type :bio.../foo ...])`

I don't see any performance related issues by appliying this approach.

And, since this is a clean up PR, it's a good opportunity to properly redefine the structure.

Reply:
I agree with @ulisesmac, nesting helps with organisation and a flat/flatter db would be more difficult to grasp and work with.

Reply:
Yeah, I strongly agree with Ulises here. The structure in activity centre and wallet is much cleaner and clearer to understand. When I first worked with communities it was extremely unclear of what the structure is and what is a root key and what is not. 
Also I noticed a lot in this pr that we talked about how consistency is key to keeping the codebase clean and understandable yet this file/topic seems to have no clear guidance and breaks that thought process.
Imo we should revisit and start working towards one root key per sub. 
cc @cammellos - I seen recently you discussing this, albeit I think for the opposite. Is there some way we can at least improve on the current approach that we have, as imo it is quite unclear of what that approach is.

Reply:
added in [3fa2426](https://github.com/status-im/status-mobile/pull/18756/commits/3fa2426b5f0ddfa482a205d9d6c21176018adc51)

decided to not move `:auth-method` as it seems not biometric specific. I guess with keycard, it would be a separate auth-method

Reply:
Good points @ulisesmac.

I would wait for @cammellos, because I remember I more strongly suggested grouping state in a PR I was reviewing under a single root key (can't find the conversation in GH though) and given his perspective made sense to me, I remember I retracted from further trying to make any guideline about this subject.

Reply:
moved this to a discussion -> https://github.com/status-im/status-mobile/discussions/18904

Please move the conversation there and feel free to update the description in that if the problem can be better defined.